pkg/devspace/pipeline/pipeline.go

```diff
@@ -223,6 +223,10 @@ func (p *pipeline) StartNewDependencies(ctx devspacecontext.Context, dependencie
 			continue
 		}
 
+		err := ensureNamespace(ctx, dependency.DependencyConfig().Namespace)
+		if err != nil {
+			return errors.Wrapf(err, "cannot run dependency %s", dependency.Name())
+		}
```

Comment:
What would you think of moving this into `startNewDependency()`? It might be out of place here, and moving it would allow for concurrent and sequential execution, but I'm not sure that matters either way 🤷

Reply:
> What would you think of moving this into startNewDependency()?

Right, seems to be the more appropriate position :+1:  thanks!